src/main/java/steps/VehicleTechnicalRecordsSteps.java

```diff
@@ -46,7 +46,9 @@ public void getVehicleTechnicalRecordsByPartialVimAndStatus(String searchIdentif
 
     @Step
     public void statusCodeShouldBe(int statusCode) {
-        response.then().statusCode(statusCode);
+        response.then()
+                .log().all()
```

Comment:
This won't put bearer token info into the log will it?

Reply:
this is logging the response, there is no token on the response